Resources/doc/cookbook/override_ldap-manager.md

```diff
@@ -56,11 +56,11 @@ fr3d_ldap:
 namespace Acme\DemoBundle\Ldap;
 
 use FR3D\LdapBundle\Ldap\LdapManager as BaseLdapManager;
-use FR3D\LdapBundle\Model\LdapUserInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
 
 class LdapManager extends BaseLdapManager
 {
-    protected function hydrate(LdapUserInterface $user, array $entry)
+    protected function hydrate(UserInterface $user, array $entry)
```

Comment:
This is not correct. Typehint does not match parent signature. Also `setDn()` is used in the function